src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final char DELIMETER = ',';
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, readFromFile(fromFileName));
+    }
+
+    private void writeToFile(String toFileName, String report) {
+        int sumBuy = Integer.parseInt(report.substring(0, report.indexOf(',')));
+        int sumSupply = Integer.parseInt(report.substring(report.indexOf(',') + 1));
+        StringBuilder stringBuilder = new StringBuilder();
+        File file = new File(toFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(stringBuilder.append(("supply,")).append(sumSupply)
+                    .append(System.lineSeparator())
+                    .append("buy,").append(sumBuy).append(System.lineSeparator())
+                    .append("result,").append(sumSupply - sumBuy).toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write to file", e);
+        }
+    }
+
+    private String readFromFile(String fromFileName) {
 
+        int sumBuy = 0;
+        int sumSupply = 0;
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            File file = new File(fromFileName);
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            String workingString = bufferedReader.readLine();
+            while (workingString != null) {
+                if (workingString.substring(0, workingString.indexOf(DELIMETER))
+                        .equals("supply")) {
+                    sumSupply += Integer.parseInt(
+                            workingString.substring(workingString.indexOf(DELIMETER) + 1));
+                } else {
+                    sumBuy += Integer.parseInt(
+                            workingString.substring(workingString.indexOf(DELIMETER) + 1));
+                }
+                workingString = bufferedReader.readLine();
+            }
+            bufferedReader.close();
```

Comment:
> let's better use try-with-resources

what about this comment?

Reply:
i changed in 1 method and missed another